index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
```

Comment:
```suggestion
  </tr>It would be great to add a button that allows players to reset the game without having to refresh the page. This could enhance user engagement by making it easier to start a new game.
```